optimas/explorations/base.py

```diff
@@ -228,13 +441,12 @@ def _load_history(
         assert history is None or isinstance(
             history, np.ndarray
         ), "Type {} not valid for `history`".format(type(history))
-        # Incorporate history into generator.
+        # Incorporate history into exploration.
         if history is not None:
-            self.generator.incorporate_history(history)
```

Comment:
It looks like this is the only place where `incorporate_history` is being used ; should this function be removed from the code.